plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -18,6 +19,13 @@ class ProductCollection extends AbstractBlock {
 	 */
 	protected $block_name = 'product-collection';
 
+	/**
+	 * An array keyed by the name of the collection containing handlers for implementing custom collection behavior.
+	 *
+	 * @var array
+	 */
```

Comment:
Nit: Just a random suggestion while going through the code. Feel free to ignore 😅

```suggestion
		/**
	 * An associative array of collection handlers.
	 *
	 * @var array<string, callable> $collection_handler_store
	 * Keys are collection names, values are callable handlers for custom collection behavior.
	 */
```